extensions/sql-database-projects/src/common/utils.ts

```diff
@@ -395,3 +396,10 @@ try {
 export function getAzdataApi(): typeof azdataType | undefined {
 	return azdataApi;
 }
+
+export async function createFolderIfNotExist(folderPath: string): Promise<void> {
+	const folderExists = await fse.exists(folderPath);
```

Comment:
Typically it's slightly better to try and create the folder and then just no-op if it fails - saves the exists call.